e2e/testdata/porch/rpkg-clone/config.yaml

```diff
@@ -11,9 +11,7 @@ commands:
       - rpkg
       - clone
       - --namespace=rpkg-clone
-      - https://github.com/platkrm/test-blueprints.git
-      - --directory=basens
-      - --ref=basens/v1
+      - https://github.com/platkrm/test-blueprints.git/basens@basens/v1
```

Comment:
Updated this test to demonstrate the new inferring behavior. There is another similar command in `rpkg-edit/config.yaml` that uses the `--directory` and `--ref` flags which I haven't modified,  demonstrating that the user can still use the flags when needed.